indico/modules/events/registration/templates/display/registration_summary.html

```diff
@@ -33,7 +48,7 @@
             {% endif %}
         </div>
         <div class="toolbar">
-            {% if registration.registration_form.modification_mode.name != 'not_allowed' %}
+            {% if registration.registration_form.modification_mode.name != 'not_allowed' and registration.is_active %}
```

Comment:
Do we also enforce this on the server side? Or would an edit go through if you sent the right request (easiest way to test: start editing while the registration is active, withdraw in another tab, then try saving the edit).

Reply:
Implicitly, it was like this already. This view could be accessed only for active registrations, but `RHRegistrationDisplayEdit` did not check if the registration is active, so someone could have potentially edited their non-active registration. I just added a `_check_access` that checks if `registration.is_active` here and in `RHRegistrationWithdraw`.